mondey_backend/src/mondey_backend/models/questions.py

```diff
@@ -52,3 +52,40 @@ class UserQuestionAdmin(SQLModel):
     input: str
     options: str
     text: dict[int, UserQuestionText] = {}
+
+
+# Answers to user questions. Internal model and 'public' model exposed to the forntend app
+
+
+class UserAnswer(SQLModel, table=True):
+    """
+    Internal model for user answers.
+
+    Parameters
+    ----------
+    UserAnswerBase : Base type for all UserAnswer models
+
+    table : bool, True
+        Makes sure this is created as a table in the database, by default True
+    """
+
+    id: int | None = Field(default=None, primary_key=True)
+    user_id: int
+    question_id: int = Field(default=None, foreign_key="userquestion.id")
+    answer: str
+    # flag that tells the frontend if the answer has been given via an  additional text field => won´t be correctly displayed otherwise
+    non_standard: bool
+
+
+class UserAnswerPublic(SQLModel):
+    """
+    External data model for UserAnswers
+
+    Parameters
+    ----------
+    SQLModel : Pydantic model  basic sqlmodel pydantic type
+    """
+
+    answer: str
```

Comment:
we could consider adding `id` here - it would make the update logic it more robust (see below)

Reply:
changed accordingly as discussed yesterday